rwkv/chat_with_bot.py

```diff
@@ -42,6 +41,7 @@
 
 parser = argparse.ArgumentParser(description='Provide terminal-based chat interface for RWKV model')
 parser.add_argument('model_path', help='Path to RWKV model in ggml format')
+parser.add_argument('tokenizer', help='Which tokenizer to use', nargs='?', type=str, default="20b")
```

Comment:
```suggestion
parser.add_argument('tokenizer', help='Which tokenizer to use', nargs='?', type=str, default="20B")
```
Upper-case letter is used in tokenizer file name and previous version of the code, let's keep it